qos/cosmos/response_health.go

```diff
@@ -1,90 +1,74 @@
 package cosmos
 
 import (
-	"encoding/json"
-
-	"github.com/pokt-network/poktroll/pkg/polylog"
-
+	"github.com/buildwithgrove/path/gateway"
 	qosobservations "github.com/buildwithgrove/path/observation/qos"
+	"github.com/buildwithgrove/path/qos"
 	"github.com/buildwithgrove/path/qos/jsonrpc"
+	"github.com/pokt-network/poktroll/pkg/polylog"
 )
 
-// responseToHealth provides the functionality required from a response by a requestContext instance.
-var _ response = responseToHealth{}
+// responseValidatorHealth implements jsonrpcResponseValidator for /health endpoint
```

Comment:
Not a blocker and this should be done in a subsequent PR but just making a note that we should likely clarify that certain methods are CometBFT vs Cosmos SDK vs EVM JSON-RP (eg. `/health` is specifically CometBFT).